Dockerfile.bak

```diff
@@ -7,8 +7,18 @@ RUN apk add --update alpine-sdk openssl-dev
 
```

Comment:
This is the original Dockerfile. Please do not modify upstream Dockerfiles. I recommend renaming your fuzzing Dockerfile to something like `Dockerfile.mayhem`, and referencing the Dockerfile in your mayhem.yml, and reverting the changes to the original Dockerfile.

Reply:
I have removed the Dockerfile.bak and restored it back to prod.Dockerfile in the docker folder. Now my own file is named Dockerfile.
.